src/middlewared/middlewared/plugins/service_/services/ups.py

```diff
@@ -21,6 +21,10 @@ async def start(self):
             await self._systemd_unit("nut-driver-enumerator", "start")
         await self._unit_action("Start")
 
+    async def after_start(self):
+        # Reconfigure mdns (add nut service)
+        await self.middleware.call('service.reload', 'mdns')
```

Comment:
`service.*` related endpoints propagate the service action to the other controller on HA systems by default. Do we need to do that with this service?

Reply:
No